chronicler/events/core/git.py

```diff
@@ -32,6 +35,36 @@
 GIT_EVENT_ACTION_COPIED = "org.grimoirelab.events.git.file.copied"
 GIT_EVENT_ACTION_TYPE_CHANGED = "org.grimoirelab.events.git.file.typechanged"
 
+GIT_EVENT_COMMIT_AUTHOR = "org.grimoirelab.events.git.author"
+GIT_EVENT_COMMIT_COMMITTER = "org.grimoirelab.events.git.committer"
```

Comment:
To keep them aligned with the other events I suggest this change:

```suggestion
GIT_EVENT_COMMIT_AUTHORED_BY = "org.grimoirelab.events.git.commit.authored_by"
GIT_EVENT_COMMIT_COMMITTED_BY = "org.grimoirelab.events.git.commit.committed_by"
```